### ISSUE TITLE
dataset.setColors not working in  LineChart

### ISSUE THREAD

**Issue by goodcity** [AppDevNext/MPAndroidChart#69]

**Comment by hannesa2** [comment]
Wenn it works on upstream, then I merged on pull request too much 😀
Btw, code is available and debugging should be not that difficult, even for you

**Comment by Paget96** [comment]
> _No description provided._

Yeah, this don't fork for quite long, I'm using a temporary fix for that.

**Comment by hannesa2** [comment]
Please see https://github.com/AppDevNext/MPAndroidChart/issues/71#issuecomment-1359005381

**Comment by hannesa2** [comment]
Please show me your code how to set colors on your side 
@goodcity

**Comment by hannesa2** [comment]
Here I set the color to blue and it's working properly https://github.com/AppDevNext/AndroidChart/pull/629 with a screenshot

I close this here, but if you have still issues, please reopen and demonstrate how the reproduce it on my side
